ingestion/tests/utils/docker_service_builders/test_container_builder.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright 2024 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""test container builder class"""
+
+from typing import List
+
+from .abstract_test_container import AbstractTestContainer
+from .database_container.mysql_test_container import MySQLTestContainer
+from .database_container.postgres_test_container import PostgresTestContainer
+
+
+class ContainerBuilder:
+    def __init__(self) -> None:
+        self.containers: List[AbstractTestContainer] = []
+
+    def build_mysql_container(self):
```

Comment:
since this is docker, I would suggest refraining from using the term `build` and use something like `deploy`,`run` or `define` depending on context. I think in the near future we'll be actually building images and we'll want to reserve the keyword for when we do..